sympy/polys/domains/domain.py

```diff
@@ -875,6 +875,8 @@ def poly_ring(self, *symbols, order=lex):
     def frac_field(self, *symbols, order=lex):
         """Returns a fraction field, i.e. `K(X)`. """
         from sympy.polys.domains.fractionfield import FractionField
+        symbols = tuple(1/symbol if not sympify(symbol).is_Number and \
+                  symbol.as_numer_denom()[0] == 1 else symbol for symbol in symbols)
```

Comment:
If this is just checking for `Pow(symbol, -1)` then I think that the check should be more direct.

Reply:
The check should be more like
```python
    if isinstance(symbol, Pow):
         ...
```

Reply:
That makes more sense that the current implementation , i'll get this fixed in the next commit !

Reply:
I think that `symbols` are actually expected to be Symbols. So if `FractionField` is modified, it could be checked that `all(s.is_Symbol for s in symbols) `.

Reply:
Cool I'll take care of this !